config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  resources :errors, only: [:show]
```

Comment:
Instead of doing this, maybe do this:
```ruby
resources :errors, only: [:show], constraints: { id: /\d{3}/ }
```
Note that now the `status` parameter has been renamed to id (to follow Rails conventions on the show parameter).